### ISSUE TITLE
Resolving webpack aliases

### ISSUE THREAD

**Issue by Tux1** [TrySound/postcss-easy-import#12]
Hi,

I cannot resolve scss file with webpack alias. is it a normal behavior ?

I get this error :

> WARNING in ./~/css-loader?{"sourceMap":true,"modules":true,"localIdentName":"[name]_[local]_[hash:base64:3]","minimize":false}!./~/postcss-loader!./components/UserPage/styles.scss
> postcss-import: /Users/toto/web/components/UserPage/styles.scss:1:1: Cannot find module 'scss/main.scss' from '/Users/toto/web/components/UserPage'

`styles.css` contains : 
`@import "scss/main.scss`
and below is the function I use in my webpack config:

```
  postcss(bundler) {
    return {
      plugins: [
        require('postcss-easy-import')({
          addDependencyTo: bundler,
          // path: path.resolve(__dirname, 'assets/scss'),
        }),
        require('autoprefixer')(),
      ],
      syntax,
      parser: syntax,
    };
```

**Comment by simonsmith** [comment]
Hi @Tux1,

Do you think you could update this issue with a full webpack config that reproduces the issue? I think the latest version of `postcss-import` resolved some webpack issues but it'd be nice to have a test case.

Thanks

**Comment by Tux1** [comment]
Hi Simon,
Will test asap 
Thanks !

**Comment by simonsmith** [comment]
Just published `2.0.0`. It now uses postcss-import 9.x which did resolve some webpack issues. Please let me know if you have further problems related to this plugin and we can reopen.
